.azure/pipelines/azure-pipelines-external-release.yml

```diff
@@ -104,19 +104,12 @@ jobs:
             }
         ]
 
-  - task: CopyFiles@2
-    displayName: 'Copy Files for Nuget package to Artifacts dir: $(build.artifactstagingdirectory)'
-    enabled: True
-    inputs:
-      Contents: '**/bin/AnyCPU/$(BuildConfiguration)/**/*'
-      TargetFolder: $(build.artifactstagingdirectory)
-
   - task: NuGetCommand@2
     displayName: nuget pack Garnet
     enabled: True
     inputs:
       command: custom
-      arguments: pack Garnet.nuspec -OutputDirectory $(Build.ArtifactStagingDirectory) -Properties Configuration=$(BuildConfiguration) -Symbols -SymbolPackageFormat snupkg -version $(Build.BuildNumber) -Verbosity Detailed
```

Comment:
we need to verify that the nupkg produced does indeed use Release build, and they are signed (by checking the contents of the .nupkg).

Reply:
1) Confirmed DLLs in nuget package are signed - looking at properties of file
2) Verified files are release and double checked output of build:
MSBuild version 17.9.8+b34f75857 for .NET
  Determining projects to restore...
  Restored /home/vsts/work/1/s/libs/common/Garnet.common.csproj (in 1.49 sec).
  Restored /home/vsts/work/1/s/libs/client/Garnet.client.csproj (in 1.49 sec).
  Restored /home/vsts/work/1/s/metrics/HdrHistogram/HdrHistogram.csproj (in 5 ms).
  Restored /home/vsts/work/1/s/libs/storage/Tsavorite/cs/src/core/Tsavorite.core.csproj (in 5 ms).
  HdrHistogram -> /home/vsts/work/1/s/metrics/HdrHistogram/bin/Release/net8.0/HdrHistogram.dll
etc
etc